ext/curl/tests/curl_setopt_CURLOPT_FOLLOWLOCATION.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+CURLOPT_FOLLOWLOCATION values
+--EXTENSIONS--
+curl
+--SKIPIF--
+<?php
+include 'skipif-nocaddy.inc';
+if (curl_version()['version_number'] < 0x080d00) die('skip requires curl >= 8.13.0');
+if (ini_get('open_basedir')) exit("skip open_basedir is set");
```

Comment:
Why do you check for the open_basedir option?

Reply:
not really intended I copied the header from another test.